src/Renci.SshNet/Sftp/SftpFileAttributes.cs

```diff
@@ -397,8 +435,8 @@ private SftpFileAttributes()
 
         internal SftpFileAttributes(DateTime lastAccessTime, DateTime lastWriteTime, long size, int userId, int groupId, uint permissions, IDictionary<string, string> extensions)
         {
-            LastAccessTime = _originalLastAccessTime = lastAccessTime;
-            LastWriteTime = _originalLastWriteTime = lastWriteTime;
+            LastAccessTimeUtc = _originalLastAccessTimeUtc = ToUniversalTime(lastAccessTime);
```

Comment:
I don't see why you'd need to convert to UTC here.
I propose to rename the arguments from **last\*Time** to **last\*TimeUtc**.

Reply:
Done.